shared/TestStandComm/SerialTransport.h

```diff
@@ -4,25 +4,33 @@
 #include "SerialDevice.h"
 #include "Messages.h"
 
-typedef enum {
-    MSG_SEG_START    = 0,
-    MSG_SEG_ID       = 1,
-    MSG_SEG_LENGTH   = 2,
-    MSG_SEG_DATA     = 3,
-    MSG_SEG_CRC      = 4,
-    MSG_SEG_END      = 5
-} MessageSegment;
-
-typedef struct {
-    MessageSegment current_segment;
-    uint8_t bytes_read;
-    uint8_t msg_length;
-    uint16_t crc;
-} PendingMessage;
-
+/**
+ * @class SerialTransport
+ * 
+ * @brief Transport layer of the serial communication protocol
+ * 
+ * The layer is aware of "Messages" (distinct packets of data with a defined structure)
+ * and is responsible for sending and receiving entire Messages.
+ */
 class SerialTransport
 {
     private:
+        typedef enum {
+            MSG_SEG_START    = 0,
+            MSG_SEG_ID       = 1,
+            MSG_SEG_LENGTH   = 2,
+            MSG_SEG_DATA     = 3,
+            MSG_SEG_CRC      = 4,
+            MSG_SEG_END      = 5
+        } MessageSegment;
+
+        typedef struct {
+            MessageSegment current_segment;
+            uint8_t bytes_read;
+            uint8_t msg_length;
```

Comment:
I remember with the motor counts values, it had to be 16 bits to prevent the values from wrapping around. I know you fixed it somewhere but just wanted to bring up in case `msg_length` needs to be longer.

Reply:
`msg_length` is a uint8_t meaning it can have any value between 0 and 255, meaning we can send a message up to 255 bytes long which is plenty (A motor move command sends like 14 bytes).

Reply:
Do you remember the spot where the size needed to be increased to accommodate for the encoder values? Just wanna check it out.

Reply:
All of the motor-related numbers are 32-bit and have been updated everywhere except feArduino (but I updated that in my demo branch will I'll make a PR for soon)